lib/ansible/modules/system/pamd.py

```diff
@@ -384,7 +384,7 @@ def write(self):
             self.fname = self.path + "/" + self.name
         # If the file is a symbollic link, we'll write to the source.
         pamd_file = os.path.realpath(self.fname)
-        temp_file = "/tmp/" + self.name + "_" + time.strftime("%y%m%d%H%M%S")
+        temp_file = os.path.join(self.ansible.tmpdir, self.name + "_" + time.strftime("%y%m%d%H%M%S"))
```

Comment:
This may cause issues similar to https://github.com/ansible/ansible/issues/38672. I mean when this module is executed with `become_user`, it may not have enough permissions to create the directory inside `self.ansible.tmpdir`).